scripts/build/complay.py

```diff
@@ -190,7 +190,7 @@ def startElement(self, name, attrs):
                     self.handleError('Element mamelayout missing attribute version')
                 else:
                     try:
-                        long(attrs['version'])
+                        int(attrs['version'])
```

Comment:
There's a better way to deal with this, see 1eb3ffe15c89f616713b45c0eee837e293a9764c